api/src/main/java/org/pmiops/workbench/api/ConceptsController.java

```diff
@@ -0,0 +1,58 @@
+package org.pmiops.workbench.api;
+
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import org.pmiops.workbench.cdr.dao.ConceptService;
+import org.pmiops.workbench.cdr.model.Concept;
+import org.pmiops.workbench.exceptions.BadRequestException;
+import org.pmiops.workbench.model.ConceptListResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Slice;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ConceptsController implements ConceptsApiDelegate {
+
+  private static final Integer DEFAULT_MAX_RESULTS = 20;
+  private static final int MAX_MAX_RESULTS = 1000;
+
+  private final ConceptService conceptService;
+
+  private static final Function<Concept, org.pmiops.workbench.model.Concept> TO_CLIENT_CONCEPT =
```

Comment:
nit: we usually import the client models, and full-qualify DB models in our other controllers

Reply:
Done.